.gitignore

```diff
@@ -58,6 +58,7 @@ docs/_build/
 troposphere.key
 troposphere/settings/local.py
 troposphere/assets
+troposphere/tropo-static
```

Comment:
For generating and hosting static files, served through runsslserver on troposphere..